_includes/configuration/contextmenu.md

```diff
@@ -27,4 +27,4 @@ tinymce.init({
 });
 ```
 
-Further examples of the `contextmenu` option are available in the [context menu examples]({{site.baseurl}}/ui-components/contextmenu/#liveexample).
\ No newline at end of file
+For information on configuring the `contextmenu` option and creating custom context menu items [context menu examples]({{site.baseurl}}/ui-components/contextmenu/).
```

Comment:
Need to either:
- wrap in a conditional
- change the link

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1495/commits/34f297c361e6e7ab7c3eb5b534e236056ed6fcf4